src/components/svg/tests/LinkSvg.spec.tsx

```diff
@@ -55,6 +55,15 @@ describe('<LinkSvg>', () => {
             expect(linkSvgComponent.find(`a[href="${linkSvgProps.url}"]`).length).toBe(1);
         });
 
+        it('should set the onClick on the anchor element', () => {
+            const onClick = jasmine.createSpy('onClick');
+
+            renderLinkSvg({onClick});
+            linkSvgComponent.find('a').prop('onClick')(undefined);
+
+            expect(onClick).toHaveBeenCalledTimes(1);

```

Comment:
Can you simulate a click on the element to make sure it works? Since the page will change, some code might not run. I think you need to prevent default and once onClick is completed you have to manually navigate